src/modules/report/base.py

```diff
@@ -22,13 +23,41 @@ def get_services(self):
         services_lock.release()
         return services_data
 
+    def get_severity(self, category):
+        severity = {
+            "Information Disclosure": "medium",
+            "Denial of Service": "medium",
+            "Remote Code Execution": "high",
+            "Identity Theft": "high",
+            "Access Risk": "low"
+        }
+        return severity.get(category, "low")
+
     def get_vulnerabilities(self):
         vulnerabilities_lock.acquire()
         vulnerabilities_data = [{"location": vuln.location(),
                  "category": vuln.category.name,
+                 "severity": self.get_severity(vuln.category.name),
                  "vulnerability": vuln.get_name(),
                  "description": vuln.explain(),
                  "evidence": str(vuln.evidence)}
                 for vuln in vulnerabilities]
         vulnerabilities_lock.release()
         return vulnerabilities_data
+
+    def get_hunter_statistics(self):
```

Comment:
Does this duplicate functionality that is already covered [here]?(https://github.com/aquasecurity/kube-hunter/blob/f0160e2eba5f87fd0c5ffdd1681f6eb71dfeaeaf/kube-hunter.py#L99)